var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -15,42 +15,45 @@ class Meme(AutotoolsPackage):
     homepage = "https://meme-suite.org"
     url = "http://meme-suite.org/meme-software/5.1.1/meme-5.1.1.tar.gz"
 
+    version("5.5.4", sha256="cda6011c2b855bf2563c4e7a2c255e11e99b5b6e5e73736ff008942507580153")
     version("5.3.0", sha256="b2ddec9db972fcf77b29c7deb62df8b1dd8a6638c13c1aa06a5d563c4a7ff756")
     version("5.2.0", sha256="0cbf8c2172e9b6c07855b8aeec457f4825f0b132f8cbb11192880e2f6033f54f")
     version("5.1.1", sha256="38d73d256d431ad4eb7da2c817ce56ff2b4e26c39387ff0d6ada088938b38eb5")
     version("4.12.0", sha256="49ff80f842b59d328588acfcd1d15bf94c55fed661d22b0f95f37430cc363a06")
     version("4.11.4", sha256="3e869ff57e327a9c8615dbef784e3f1095f7f7a0120cecd55efe10c3f2ee8eb3")
 
     variant("mpi", default=True, description="Enable MPI support")
-    variant("image-magick", default=False, description="Enable image-magick for png output")
+    variant("imagemagick", default=False, description="Enable imagemagick for png output")
```

Comment:
FWIW. You could shorten it to `magic` as I've seen in the only other package (`octave`) using a variant for `imagemagick`.

Reply:
Sure, shortened it down to `magick`. No hyphen in `ImageMagick`, so was just neatening things up before.